src/control/run_go_tests.sh

```diff
@@ -56,26 +56,42 @@ function setup_environment()
 	CGO_CFLAGS+=" -I${SL_HWLOC_PREFIX}/include"
 }
 
+function check_formatting()
+{
+	srcdir=${1:-"./"}
+	output=$(find $srcdir/ -name *.go -and -not -path '*vendor*' \
```

Comment:
(lint) Use -print0/-0 or -exec + to allow for non-alphanumeric filenames. [SC2038]
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Quote the parameter to -name so the shell won't interpret it. [SC2061]
(lint) Use ./*.go so names with dashes won't become options. [SC2035]